app/components/about/about.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module About
+  class About < ApplicationComponent
+    private
+
+    def version
+      Setting.git_commit_sha[0, 8]
```

Comment:
```suggestion
      Setting.git_commit_sha
```
Why shortened? :shrug:

Reply:
I find commit hashes a little easier to skim when they are shortened (especially as they are usually displayed shortened in tools like GitHub as well), especially as for most projects it’s very unlikely that there are collision with 8+ characters.